projects/abtesting/src/app/core/experiment-users/store/experiment-users.model.ts

```diff
@@ -3,7 +3,7 @@ import { EntityState } from '@ngrx/entity';
 
 export enum EntityTypes {
   GROUP_ID = 'Group ID',
-  USER_ID = 'User ID'
+  PARTICIPANT_ID = 'PARTICIPANT ID'
```

Comment:
@tanmayjain3 Shouldn't it be Participant ID instead of PARTICIPANT ID?

Reply:
Updated